apps/web-tools/trustchain/components/AppAccountsSync.tsx

```diff
@@ -36,12 +42,16 @@ import { inferCryptoCurrencyIcon } from "@ledgerhq/live-common/currencies/crypto
 
 type State = {
   accounts: Account[];
+  nonImportedAccounts: NonImportedAccountInfo[];
   walletState: WalletState;
 };
```

Comment:
You use this type multiple times, you can store it in a file and reuse it everywhere it's needed? :)

Reply:
not sure to follow, where is it used? it's the web-tools here, right?